lib/modern_searchlogic/column_conditions.rb

```diff
@@ -84,8 +92,12 @@ def searchlogic_suffix_condition_match(method_name)
             arity: arity,
             block: lambda do |*args|
               validate_argument_count!(arity, args.length) if arity >= 0
-              arel_conditions = column_names.map { |n| instance_exec(n, *args, &method_block) }.reduce(:or)
-              where(arel_conditions)
+              condition = column_names.map { |n| instance_exec(n, *args, &method_block) }.reduce(:or)
```

Comment:
I think there's an additional path above that Rails 3 and Rails 4 won't know how to handle, e.g.:

```ruby
User.username_or_email_in("foo", "foo@example.com")
```

ActiveRecord 3 and 4 don't know how to `:or` now that ActiveRecord handles `in` and `not_in` it will blow up methinks on the `reduce(:or)` above.